master_project/itemPose[] item_pose_list